src/modules/Security.rb

```diff
@@ -517,14 +534,18 @@ def write_to_locations
       end
     end
 
-    # Write settings related to PAM behavior
-    def write_pam_settings
-      # pam stuff
-      encr = @Settings.fetch("PASSWD_ENCRYPTION", default_encrypt_method)
-      if encr != @Settings_bak["PASSWD_ENCRYPTION"]
-        SCR.Write(path(".etc.login_defs.ENCRYPT_METHOD"), encr)
+    # Write login.defs configuration
+    def write_shadow_config
+      SHADOW_ATTRS.each do |attr|
+        shadow_config.public_send("#{attr.to_s.downcase}=", @Settings[attr])
       end
+      encr = @Settings.fetch("PASSWD_ENCRYPTION", default_encrypt_method)
```

Comment:
For parity, I'd have a `write_encryption_method` method. But I understand that it implies calling `#save` twice.

Reply:
Yes, there is some asymmetry there.